.changeset/kind-vans-add.md

```diff
@@ -0,0 +1,7 @@
+---
+"@shopify/shopify-api": patch
+---
+
+Added an `is_default` field to `CustomerAddress` so it doesn't overlap with the existing `default()` method we provide in the class.
```

Comment:
Let's add an example to clarify

To access the default boolean field of the CustomerAddress
```
const customer = new shopify.rest.CustomerAddress({session});
console.log(customer.is_default) // false
```